relayer.go

```diff
@@ -38,6 +40,11 @@ type Relayer interface {
 
 type MetaTxnID string
 
+var (
```

Comment:
FYI, we have these computed here: https://github.com/0xsequence/go-sequence/blob/master/transactions.go#L94